$NetBSD$

Only common data installation directories are needed for these protocols.
Avoid querying Qt tools for an XML and CMake data package.

--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -14,7 +14,7 @@ find_package(ECM 6.0.0 NO_MODULE)
 set_package_properties(ECM PROPERTIES TYPE REQUIRED DESCRIPTION "Extra CMake Modules." URL "https://commits.kde.org/extra-cmake-modules")
 feature_summary(WHAT REQUIRED_PACKAGES_NOT_FOUND FATAL_ON_MISSING_REQUIRED_PACKAGES)
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake/Modules)
-include(KDEInstallDirs)
+include(KDEInstallDirsCommon)
 include(KDECMakeSettings)
 include(ECMSetupVersion)
 
